src/status_im/ui/screens/wallet/recipient/views.cljs

```diff
@@ -0,0 +1,286 @@
+       (when @opened?
+         content)])))
+
+(defn render-account [account]
+  [quo/list-item
+   {:icon     [chat-icon/custom-icon-view-list (:name account) (:color account)]
+    :title    (:name account)
+    :on-press #(re-frame/dispatch [:wallet.send/set-recipient (:address account)])
+    :subtitle [quo/text {:monospace true
+                         :color     :secondary}
+               (utils/get-shortened-checksum-address (:address account))]}])
+
+(def scroll-view-ref (atom nil))
+
+(defn contacts-list-item [{:keys [name] :as contact}]
+  (let [[first-name second-name] (multiaccounts/contact-two-names contact true)]
+    [quo/list-item
+     {:title    first-name
+      :subtitle second-name
+      :on-press #(do
+                   (some-> ^js @scroll-view-ref (.scrollTo #js {:x 0 :animated true}))
+                   (re-frame/dispatch [:wallet.recipient/address-changed name]))
+      :icon     [chat-icon/contact-icon-contacts-tab
+                 (multiaccounts/displayed-photo contact)]}]))
+
+(defn empty-items [icon title]
+  [react/view {:height 94 :align-items :center :justify-content :center}
+   [icons/icon icon
+    {:color colors/gray}]
+   [react/text {:style {:color colors/gray :margin-top 8}}
+    title]])
+
+(views/defview accounts-section []
+  (views/letsubs [accounts [:accounts-for-recipient]]
+    (let [cnt (count accounts)]
+      [section
+       (i18n/label :t/my-accounts)
+       cnt
+       (if (> cnt 0)
+         [react/view
+          (for [account accounts]
+            [render-account account])]
+         [empty-items :main-icons/address (i18n/label :t/my-accounts-empty)])])))
+
+(defn render-recent [{:keys [from to type amount-text currency-text]}]
+  (let [inbound? (= type :inbound)
+        address (if inbound? from to)]
+    [quo/list-item
+     {:title     [quo/text {:monospace true}
+                  (utils/get-shortened-checksum-address address)]
+      :on-press  #(re-frame/dispatch [:wallet.recipient/address-changed address])
+      :size      :small
+      :accessory [react/text {:style {:flex-shrink 1
+                                      :color       colors/gray}}
+                  (str (if inbound? "↓ " "↑ ") amount-text " " currency-text)]}]))
+
+(defn recent-section []
+  (let [{:keys [from]} @(re-frame/subscribe [:wallet/prepare-transaction])
+        txs @(re-frame/subscribe [:wallet/recipient-recent-txs (:address from)])
+        cnt (count txs)]
+    [section
+     (i18n/label :t/recent)
+     cnt
+     (if (> cnt 0)
+       [react/view
+        (for [{:keys [data]} txs]
+          [render-recent (first data)])]
+       [empty-items :main-icons/history (i18n/label :t/recent-empty)])]))
+
+(defn render-fav [{:keys [address name]}]
+  (let [noname? (string/blank? name)
+        short-address (utils/get-shortened-checksum-address address)]
+    [quo/list-item
+     {:title    (if noname? [quo/text {:monospace true}
+                             short-address] name)
```

Comment:
can we move `name` to the next line here? imo this formatting makes it a bit hard to read

Reply:
sure